src/actions/ServiceGraphActions.ts

```diff
@@ -0,0 +1,19 @@
+import { createAction } from 'typesafe-actions';
+import { CytoscapeClickEvent } from '../components/CytoscapeGraph/CytoscapeGraph';
+
+export enum ServiceGraphActionKeys {
+  GRAPH_OBJECT_CLICKED = 'GRAPH_OBJECT_CLICKED',
+  GRAPH_READY = 'GRAPH_READY'
+}
+
+// synchronous action creators
+export const ServiceGraphActions = {
+  graphObjectClicked: createAction(ServiceGraphActionKeys.GRAPH_OBJECT_CLICKED, (event: CytoscapeClickEvent) => ({
```

Comment:
Is `graphObjectClicked` the right name for this? That is the thing that dispatches this Redux action not what the actual action does. The name should be what the redux action does.